client/my-sites/pages/page-list.jsx

```diff
@@ -202,15 +202,16 @@ class Pages extends Component {
 	 * Show the virtual homepage
 	 */
 	showVirtualHomepage() {
-		const { site, homepageType, blockEditorSettings, isFSEActive, translate } = this.props;
+		const { site, homepageType, homepageId, blockEditorSettings, isFSEActive, translate } =
+			this.props;
 		const { search, status } = this.props.query;
 
 		return (
 			config.isEnabled( 'unified-pages/virtual-home-page' ) &&
 			/** Virtual homepage is for themes that support FSE */
 			isFSEActive &&
 			site &&
-			homepageType === 'posts' &&
+			( homepageType === 'posts' || ( homepageType === 'page' && ! homepageId ) ) &&
```

Comment:
we have to update the condition in showBlogPostsPage as well

Reply:
Sure